genkit-tools/common/src/types/eval.ts

```diff
@@ -88,7 +88,7 @@ export type EvalResult = z.infer<typeof EvalResultSchema>;
  * A unique identifier for an Evaluation Run.
  */
 export const EvalRunKeySchema = z.object({
-  actionId: z.string().optional(),
+  actionRef: z.string().optional(),
```

Comment:
there are existing/saved eval runs, this change will break those right? Can you add `actionRef` without removing `actionId`?

Reply:
Yeah it will break. 

Yes, that's a good idea. That should be possible.

Reply:
Given that main is only for critical fixes, I think we can merge this newer version into `next` without complications.